app/backend/src/main/java/com/example/controllers/ItemController.java

```diff
@@ -0,0 +1,23 @@
+package com.example.controllers;
+
+import com.example.models.Item;
+import com.example.services.ItemService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/items")
+public class ItemController {
+    @Autowired
+    private ItemService itemService;
+
+    @GetMapping
+    public List<Item> getAllItems() {
```

Comment:
Naming nitpick again, no need for specifying getAll<Type>. Just name it getAll and call it a day :D 
Applies to every other name also.